custom_components/fellow/fellow_aiden/__init__.py

```
@@ -99,9 +114,18 @@ async def _request(
+            try:
+                response = await self._session.request(
+                    method, url, headers=headers, **kwargs
+                )
+            except (
+                aiohttp.ClientError,
+                asyncio.TimeoutError,
+                ssl.SSLError,
+            ) as err:
+                raise FellowConnectionError(
+                    f"Request failed for {method.upper()} {url}: {err}"
+                ) from err
```

Ruff (0.15.10)

[warning] Avoid specifying long messages outside the exception class

(TRY003)